### PR TITLE
Loginpage

### PR DESCRIPTION
**Pull request by oyayafelix**

### CODE REVIEW

File: client/src/components/loginform.js [hunk 1]
```diff
@@ -7,14 +7,14 @@ const LoginForm = ()=> {
         <div className="loginform">
             <img src={loginavi} alt="login avi" />
             <h1>Inforum</h1>
-            <h3>Give your ideas a voice.</h3>
+            <h2>Give your ideas a voice.</h2>
```
Comment (chibukasia) [inline client/src/components/loginform.js]: Replace h2 content with Log In.

File: client/src/components/loginform.js [hunk 2]
```diff
@@ -7,14 +7,14 @@ const LoginForm = ()=> {
         <div className="loginform">
             <img src={loginavi} alt="login avi" />
             <h1>Inforum</h1>
```
Comment (chibukasia) [inline client/src/components/loginform.js]: Consider using a form and have the inputs nested in a form. That would be easier for using onSubmit event handler. You can research on forms

File: client/src/components/loginform.js [hunk 3]
```diff
@@ -7,14 +7,14 @@ const LoginForm = ()=> {
...
-            <h3>Give your ideas a voice.</h3>
+            <h2>Give your ideas a voice.</h2>
             <input type="text"placeholder="Username" />
             <input type="password"placeholder="Password" />
 
             <div>
             <button type="button" className="login-btn">Continue</button>
```
Comment (chibukasia) [inline client/src/components/loginform.js]: In the form you will create. Replace this with a <input type ="submit value="Logi In">

**Review by chibukasia — CHANGES_REQUESTED** [review]
Please consider reviewing the requested changes

File: client/src/components/loginform.js (hunk 1 above)
Comment (oyayafelix) [reply client/src/components/loginform.js]: changed

File: client/src/components/loginform.js (hunk 3 above)
Comment (oyayafelix) [reply client/src/components/loginform.js]: done

File: client/src/components/loginform.js (hunk 2 above)
Comment (oyayafelix) [reply client/src/components/loginform.js]: done

**Comment by chibukasia** [comment]
Its okay now

**Review by chibukasia — APPROVED** [review]

**Pull request merged**